src/main/java/Duke.java

```diff
@@ -6,5 +9,81 @@ public static void main(String[] args) {
                 + "| |_| | |_| |   <  __/\n"
                 + "|____/ \\__,_|_|\\_\\___|\n";
         System.out.println("Hello from\n" + logo);
+        System.out.println("\t____________________________________________________________\n"
```

Comment:
Maybe you can consider the printing of the welcome message in another method for the code to be tidier, like what you did for the goodbye message. -SWJE